src/BPEssentials/Commands/Pay/Pay.cs

```diff
@@ -0,0 +1,28 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BPEssentials.Utils;
+
+using BrokeProtocol.Entities;
+using BrokeProtocol.Utility;
+using System.Linq;
+
+
+namespace BPEssentials.Commands
+{
+    public class Pay : Command
+    {
+        public void Invoke(ShPlayer player, ShPlayer target, int amount=1)
+        {
+            var item = Core.Instance.EntityHandler.Items.FirstOrDefault(x => x.Key == "Money");
+            if (player.MyItemCount(item.Value.index) < amount)
```

Comment:
By accessing the key each time you execute the command you will loop over the whole dictionary every time. A better way would've been `Items.TryGetValue(key, out T item)`. Though, the game already provides you with some handy methods for counting your money and transferring it.
```suggestion
            if (player.MyMoneyCount() < amount)
```